### DISCUSSION TITLE
IBAX başka görevler var mı?

### DISCUSSION THREAD

**Discussion by mkakenzo** [IBAX-io/go-ibax#1128] [Soru Cevap - Türkçe] [answered]
Arkadaşlar merhaba, tweet ler haricinde yapabileceğimiz başka görevler varmıdır?

**Comment by Yusufknk** [comment, marked as answer]
Çekilişler ve github ödülleri de mevcut.

**Reply by mkakenzo** [reply]
github ödüllerini nasıl yapacağımı bulamadım. ilgili link varmıdır? yada biliyorsanız yardımcı olabilirmisiniz?

**Reply by Yusufknk** [reply]
İbax Türkiye telegram grubunda sabitlenmiş mesajlarda gösterilmiştir.
